### PR TITLE
Travis ci integration added

### PR DESCRIPTION
**Pull request by shalithasuranga**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Contributing
```
Comment (dinukadesilva) [inline CONTRIBUTING.md]: @shalithasuranga 
Adding `CONTRIBUTING.md` has to be not a part of this PR since it's not relevant or a dependency to the PR.

**Review by dinukadesilva — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (shalithasuranga) [reply CONTRIBUTING.md]: Hi..
@dinukadesilva
Thanks for pointing out the issue. I removed CONTRIBUTING.md from this PR

**Comment by rehrumesh** [comment]
@uslperera Please review

**Comment by dinukadesilva** [comment]
@shalithasuranga why dont't you wrap all the commits to a one single commit with a meaningful message.

**Review by dinukadesilva — APPROVED** [review]

**Pull request merged**